documentation/creating-components/tutorial-component-composite.asciidoc

```diff
@@ -40,9 +36,16 @@ public class TextField extends Composite<Div> {
 }
 ----
 
-A Composite will automatically create the root component, specified using generics (`Composite<Div>`) and make it available through getContent(). In the constructor you only need to create the other components and add them to the root `Div`. The value is set using `setValue` in the `Input` component.
+* The `Composite` automatically creates the root component (specified using generics (`Composite<Div>`)).
+* The composite is available through the `getContent()` method. 
```

Comment:
```suggestion
* The root component is available through the `getContent()` method. 
```
The composite is not just the root content, it is the class wrapping all the building block to create the new more complex component. In this case the root is the div element wrapping Label and Input